src/Entity/ApiConfiguration.php

```diff
@@ -5,14 +5,15 @@
 namespace Synolia\SyliusAkeneoPlugin\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
-use Sylius\Component\Resource\Model\ResourceInterface;
 use Symfony\Component\Validator\Constraints as Assert;
+use Synolia\SyliusAkeneoPlugin\Config\AkeneoEditionEnum;
 
 /**
+ * @deprecated To be removed in 4.0.
```

Comment:
Same here, we should indicate what to use instead

Reply:
moving it to the next PR #118